tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,212 @@
+import { useBackend } from "../backend";
+import {
+  Box,
+  Icon,
+  Stack,
+  Button,
+  Section,
+  NoticeBox,
+  LabeledList,
+  Collapsible,
+} from "../components";
+import { Window } from "../layouts";
+
+export const Vote = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, question, lower_admin } = data;
+
+  // Adds the voting type to title if there is an ongoing vote
+  let windowTitle = "Vote";
+  if (mode) {
+    windowTitle
+      += ": " + (question || mode).replace(/^\w/, c => c.toUpperCase());
+  }
```

Comment:
Use string interpolation, AKA \`bla bla bla ${variableHere} bla \`.

Reply:
Stop, I wrote this code.